### DISCUSSION TITLE
Send Emails 📨

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#251] [Ideas]
After https://github.com/twentyhq/twenty/discussions/211 the next step will be to let the user send emails directly from the app.
This also need to be done after https://github.com/twentyhq/twenty/discussions/249 since we need to build the show/edit view first before we can implement this feature

See also: https://github.com/twentyhq/twenty/discussions/252

**Comment by Grienauer** [comment]
Would be perfect do segment users and send to the segment personalized mails out of the system.
I think this is a killer feature ;P

**Reply by etiennea** [reply]
Greed this is great if you can make groups, for single emails its not that interesting

**Reply by gvkhna** [reply]
Agreed, if this seems to be planned, sending single emails doesn't make sense. Simple email sequences would be great too. I feel like that would be more like what twenty is probably being used for, in terms of personal sales and mostly periodic follow up to warm leads etc.

**Comment by gotjoshua** [comment]
any update on the sending email situation?
WYSIWYG?
templates?

**Reply by FelixMalfait** [reply]
Still not a short-term priority sorry. Will come in 2025 for sure, I'd say around Q2 next year only.
With Workflows in December it will be able to do basic email automation but not yet something very advanced (no mass emailing, no email composer, no template, etC.)

**Comment by ganeshkrishnan1** [comment]
Is this still in the works? How do most startups that use this CRM manage the emails to their customers? Even if there is a workaround using any other tool that can automate this I am ready to use that

**Reply by BernhardBaumrock** [reply]
@ganeshkrishnan1 see this answer I just got: https://github.com/twentyhq/twenty/discussions/918#discussioncomment-13654214

**Comment by BOHEUS** [comment]
Closing in favor of planned issue https://github.com/twentyhq/core-team-issues/issues/811

**Comment by jevtih-svg** [comment]
Overall, better email app, with drafts and ability to draft emails with AI, is a must for such application. I've currently built my own mini-app to compensate for the lack of outbox/drafts and adding templates, but long term would be nice to see this native.

**Reply by FelixMalfait** [reply]
Thanks for the feedback! We're working on it

**Reply by fabiensebban** [reply]
Hi @jevtih-svg , did you publish this app you did?

**Comment by fabiensebban** [comment]
I’d like to clarify whether reusable HTML email templates are part of the planned email composer work in [#core core-team-issues#811](https://github.com/twentyhq/core-team-issues/issues/811).  
The expected functionality would be similar to [Resend](https://resend.com/) templates:
- reusable HTML templates stored in Twenty;
- dynamic variables populated from CRM records and workflow context;
- editable subject and body;
- template selection from the email composer and the “Send Email” workflow action;
- ideally previewing, versioning, and API support.
Is this feature in the roadmap?

**Reply by BOHEUS** [reply]
@neo773

**Comment by neo773** [comment]
@fabiensebban 
Hi, we will be adding first class support for HTML templates very soon. Right now, we are just polishing and adding core features like open and click tracking metrics.

**Reply by fabiensebban** [reply]
Thanks @neo773 for your response! Don't hesitate to tell me if there is any PR we can contribute.
